### PR TITLE
Feat/email template

### PR DESCRIPTION
**Pull request by colganmeanor**
Added: 

- html and CSS files for email template for creating new event

### CODE REVIEW

**Review by matthewjholmes — APPROVED** [review]
Looks great to me.

File: app/views/event_mailer/new_event_email.html.erb [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
+    <link rel="stylesheet" href="../../assets/new_event_email.css">
   </head>
   <body>
-
-    <p>You created a new event. <%= @event.name %>!</p>
+    <header>
+      <h1>Do Goodr</h1>
+    </header>
+    <div class='email-message'>
+      <p> Congratulations <%= @event.organization_name %> - your volunteering opportunity has successfully been posted to Do Goodr! The details of this event are below:</p>
```
Comment (matthewjholmes) [inline app/views/event_mailer/new_event_email.html.erb]: Will need to be `@event.organization.name`

File: app/views/event_mailer/new_event_email.html.erb [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
+    </div>
+    <div class='event-details-container'>
+      <p> Event Name: <%= @event.name %> </p>
+      <p> Address: <%= @event.address %> </p>
+      <p> Type of Event: <%= @event.category %> </p>
+      <p> Date: <%= @event.date %> </p>
+      <p> Time: <%= @event.start_time %> - <%= @event.end_time %>  </p>
```
Comment (matthewjholmes) [inline app/views/event_mailer/new_event_email.html.erb]: We may want to format this a bit more extensively to not get an unweildy DateTime.

File: app/views/event_mailer/new_event_email.html.erb [hunk 3]
```diff
@@ -1,10 +1,27 @@
...
+      <h1>Do Goodr</h1>
+    </header>
+    <div class='email-message'>
+      <p> Congratulations <%= @event.organization_name %> - your volunteering opportunity has successfully been posted to Do Goodr! The details of this event are below:</p>
+    </div>
+    <div class='event-details-container'>
+      <p> Event Name: <%= @event.name %> </p>
+      <p> Address: <%= @event.address %> </p>
+      <p> Type of Event: <%= @event.category %> </p>
+      <p> Date: <%= @event.date %> </p>
```
Comment (matthewjholmes) [inline app/views/event_mailer/new_event_email.html.erb]: No such attritribute--it's part of start_time

**Comment by colganmeanor** [comment]
I think everything should be ready to merge now @matthewjholmes - please let me know if there is anything else that needs attention. 

 - Added inline styles / deleted assets directory with css file
 - fixed slash with event date formatting
 - added font family import to reflect same font as the FE app

**Pull request merged**